src/evict/evict_lru.c

```diff
@@ -432,12 +432,6 @@ __evict_server(WT_SESSION_IMPL *session, bool *did_work)
             return (0);
         WT_RET(ret);
 
-        /*
```

Comment:
I saw these changes in the (now closed) WT-11300 PR as well. This work could be duplicating WT-12608